<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(50, 50, 60)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>-10</y>
      <width>1800</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="countDown">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>30</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
border: 2px solid white;
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>Not Recording</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>1280</width>
        <height>800</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="recordButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>RECORD</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="filename">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius: 20px;
	background: solid white;
	padding-left: 20px;
}

QLineEdit:focus{
	border: 2px solid rgb(85,170,255)
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="placeholderText">
       <string>Filename</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="downButton">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>30</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>DOWNSAMPLING</string>
      </property>
     </widget>
     <widget class="QPushButton" name="playButton">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>30</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>PLAYBACK</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="everCheckBox">
      <property name="geometry">
       <rect>
        <x>1330</x>
        <y>40</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {
	width: 18px;
	height: 18px;
	color: white;
}

QCheckBox{
	color: white;
}</string>
      </property>
      <property name="text">
       <string>everyI</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="windCheckBox">
      <property name="geometry">
       <rect>
        <x>1330</x>
        <y>70</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {
	width: 18px;
	height: 18px;
	color: white;
}

QCheckBox{
	color: white;
}</string>
      </property>
      <property name="text">
       <string>window</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="savingLabel">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>410</y>
        <width>831</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>35</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
